client/app/cloud/project/compute/infrastructure/virtualMachine/add/cloud-project-compute-infrastructure-virtualMachine-add.html

```diff
@@ -220,7 +220,7 @@
                                                 data-on-click="$ctrl.updateSshKeyRegion()"
                                                 data-ng-if="$ctrl.model.sshKey && ($ctrl.model.region.disabled === 'SSH_KEY' || (region.dataCenters.length === 1 && region.dataCenters[0].disabled === 'SSH_KEY'))">
                                     </oui-button>
-                                    <span data-translate="'cpcivm_add_step2_disabled_' + region.disabled"
+                                    <span data-text="{{ ::'cpcivm_add_step2_disabled_' + region.disabled | translate }}"
```

Comment:
same here

Reply:
there is an issue with data-translate and string concatenation. Its not working. As an alternative, I have used data-text.